test_conformance/vulkan/main.cpp

```diff
@@ -97,35 +99,47 @@ bool isDeviceSelection(const char *arg)
         || strcmp(arg, "CL_DEVICE_TYPE_DEFAULT") == 0;
 }
 
-size_t parseParams(int argc, const char *argv[], const char **argList)
+void parseParams(int &argc, const char *argv[])
 {
-    size_t argCount = 1;
-    for (int i = 1; i < argc; i++)
+    argc = parseCustomParam(argc, argv);
+
+    for (int i = 0; i < argc; ++i)
     {
+        int argsRemoveNum = 0;
+
         if (argv[i] == NULL) break;
         if (argv[i][0] == '-')
         {
             if (!strcmp(argv[i], "--debug_trace"))
             {
                 debug_trace = true;
+                argsRemoveNum = 1;
             }
             if (!strcmp(argv[i], "--useSingleImageKernel"))
             {
                 useSingleImageKernel = true;
+                argsRemoveNum = 1;
             }
             if (!strcmp(argv[i], "--useDeviceLocal"))
             {
                 useDeviceLocal = true;
+                argsRemoveNum = 1;
+            }
+            if (!strcmp(argv[i], "--useValidationLayers"))
```

Comment:
We should add this new command line argument to `printUsage`.

Actually, it looks like a bunch of command line arguments are missing in `printUsage`, and a few that are described are not actually command line arguments...

Reply:
Corrected. @nikhiljnv I've complement missing descriptions but I will appreciate your assistance, it looks like you are the author. Thanks